melodic/distribution.yaml

```diff
@@ -1237,6 +1237,21 @@ repositories:
       url: https://github.com/ros-planning/navigation_msgs.git
       version: jade-devel
     status: maintained
+  nerian_stereo:
+    doc:
+      type: git
+      url: https://github.com/nerian-vision/nerian_stereo.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
```

Comment:
there is no tag `release/melodic/*` on the release repository. It looks like you ran your melodic release using the `kinetic` track. Please try rerunnning bloom-release with `--track melodic --new-track`

Reply:
Sorry for the mistake. Just fixed it.